ee/tabby-schema/src/schema/mod.rs

```diff
@@ -287,6 +287,20 @@ impl Query {
         .await
     }
 
```

Comment:
instead of adding new interface, modify `users` to take optional email input to used as filter.